### ISSUE TITLE
clr-tabs shows ExpressionChangedAfterItHasBeenCheckedError in console

### ISSUE THREAD

**Issue by KevinABoucher** [vmware/clarity#944]
[ x] bug
[ ] feature request
[ ] enhancement

### Expected behavior
clr-tabs can exist in a popup with no error in the console

### Actual behavior
The following error is seen in the console:
ERROR Error: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'false'. Current value: 'true'.

https://plnkr.co/edit/ouRMAG?p=preview

### Reproduction of behavior
https://plnkr.co/edit/ouRMAG?p=preview

### Environment details

* **Angular version:** 4.1.3

* **Clarity version:** 0.9.5

* **OS and version:**  win10 latest

* **Browser:** Version 58.0.3029.110 (64-bit) - chrome

**Comment by rbirkgit** [comment]
This may not be a Clarity issue? Doing this removes the exception:

https://stackoverflow.com/questions/39787038/how-to-manage-angular2-expression-has-changed-after-it-was-checked-exception-w

**Comment by Igogrek** [comment]
We have the same issue if [clr-nav-level] is used outside of root html. (in our case - separate component for subnav)
```html
<nav class="sub-nav" [clr-nav-level]="1">
...
</nav>
```

**Comment by jeeyun** [comment]
Hi @KevinABoucher, we have simplified and revamped our tab API for 0.10.0. In the new version, you can achieve the same results with the following markup:

```
<clr-tabs>
    <clr-tab>
        <button clrTabLink>Settings</button>
        <clr-tab-content *clrIfActive>
            (content for settings tab)
        </clr-tab-content>
    </clr-tab>
    <clr-tab>
        <button clrTabLink>Notifications</button>
        <clr-tab-content *clrIfActive>
            <(content for notifications tab)
        </clr-tab-content>
    </clr-tab>
</clr-tabs>
```

With the use of `*clrIfActive` now the tab content only gets created when it becomes active. You can read more about it here: https://vmware.github.io/clarity/news/0.10.0-alpha.5

See this plunkr for an example of lazy loading the tab when you click on it: https://plnkr.co/edit/m4e1HKN4UsiOkP0UKCMC?p=preview

You should see in the console that the component is only created when the second tab becomes active, which I think is what you are trying to achieve. 

That plunkr is currently using the latest released clarity version `0.10.0-rc.1`. If you are weary of using the rc version, we're releasing the official `0.10.0` version this week so you can wait to upgrade then. 

Try it out and let us know if we haven't addressed your question. I'm closing this issue for now.
